change/@azure-msal-browser-1dde9e39-4138-48c1-b13d-55e395dd1c11.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Set SameSite=None on localStorage encryption cookie",
```

```suggestion
  "comment": "Set SameSite=None on localStorage encryption cookie #7549",
```
Are we still doing PR numbers?